src/features/auth/utils/get-os-variant.tsx

```
@@ -0,0 +1,35 @@
+
+  return { prefixIcon, buttonText };
+}
+
+export function getVariant(
+  os: string,
+  device: string,
+): [React.ReactNode, string] {
+  const { t } = useTranslation();
+
+  if (os.startsWith('windows')) {
+    return [<WindowsHelloIcon />, t('login.buttons.login_with_windows_hello')];
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)